ztag/schema.py

```diff
@@ -106,6 +106,31 @@ class CensysString(WhitespaceAnalyzedString):
     "timestamp":DateTime(),
 })
 
+ztag_dsa_params = SubRecord({
+    "p":IndexedBinary(),
+    "q":IndexedBinary(),
+    "g":IndexedBinary(),
+    "y":IndexedBinary(),
+})
+
+ztag_ecdsa_public_key = SubRecord({
+    "pub":IndexedBinary(),
+    "b":IndexedBinary(),
+    "gx":IndexedBinary(),
+    "gy":IndexedBinary(),
+    "n":IndexedBinary(),
+    "p":IndexedBinary(),
+    "x":IndexedBinary(),
+    "y":IndexedBinary(),
+    "curve":Enum(),
```

Comment:
I see a few other places in the schema where `Enum()` alone appears:

• [`parsed_certificate` uses it for the same property of an ECDSA public key](https://github.com/zmap/ztag/blob/ff309ab3b68f07df0cad76e204ea9c4e2ac4044b/ztag/schema.py#L187)
• it also [uses `Enum()` for its validation level](https://github.com/zmap/ztag/blob/ff309ab3b68f07df0cad76e204ea9c4e2ac4044b/ztag/schema.py#L339)

But it makes sense intuitively that an enum's values _should_ be enumerated somewhere. What does ES do with these existing fields, which don't have an explicit list of possible values?